components/wpt-results.html

```diff
@@ -194,12 +194,12 @@
             type: Boolean,
             computed: '_computeIsLatest(sha)'
           },
-          // ['browser@sha', ...] specs for the runs to load using /api/run
-          // e.g. ['chrome@latest', 'edge@latest', 'firefox@latest', 'safari@latest']
-          testRunSpecs: {
+          // URLs to load to get the TestRun JSON
+          testRunResources: {
```

Comment:
Fun fact - if this is called testRunResourceUrls instead, polymer auto-magically escapes the already-escaped string as a URL, which breaks the unmarshalling...

Reply:
Thanks for "helping" Polymer.